app/views/hyrax/collections/show.html.erb

```diff
@@ -24,7 +24,7 @@
       <div class="col-md-4 text-right">
         <%# check if institution is OSU or UO. OSU image needs 100px hieght, UO 55px %>
         <% path, alt, height = controller.institution_image_info %>
-        <%= image_tag path, alt: alt, height: height %>
+        <%= image_tag path, alt: alt, style: "max-height: #{"#{@presenter.solr_document.institution.first.to_s.include?('http://id.loc.gov/authorities/names/n80017721') ? 100 : 45 }px;"}" %>
```

Comment:
Would this set of institution image information be more appropriate here?
https://github.com/OregonDigital/OD2/blob/master/config/initializers/hacks/hyrax_collections_controller_behavior.rb#L45